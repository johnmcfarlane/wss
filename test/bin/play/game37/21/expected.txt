 43  2,10 - BIGG
 42  4,4  - BINGO
 39  2,10 - NOGG
 32 13,3  | SIB
 32 13,3  | SOB
 30 11,1  - SNIB
 30 11,1  - SNOB
 30 11,1  - SOIGN
 29  1,10 - BIGG
 27  1,10 - BING
 27  1,10 - BONG
 27 11,1  - SIGN
 27 11,1  - SING
 27 11,1  - SNIG
 27 11,1  - SNOG
 27 11,1  - SONG
 26 15,4  | CREDOS
 25  1,10 - BIOG
 25  1,10 - GING
 25  1,10 - GONG
 25  2,10 - NBG
 25  1,10 - NOGG
 25  7,13 | OBB
 24  4,2  - BINGO
 24  8,1  | OBSIGN
 24 11,1  - SIB
 24 11,1  - SOB
 23  2,10 - BIG
 23  2,10 - BOG
 23  9,13 | OBE
 22  5,4  - GOBI
 22 13,4  | IB
 22 13,4  | OB
 22  9,6  | OBS
 21  3,11 - BINGO
 21  3,11 - BONGS
 21  4,6  - GIBSONS
 21  2,10 - GIG
 21  2,10 - GOG
 21  1,10 - NIOG
 21 11,1  - SIG
 21  1,10 - SING
 21 11,1  - SION
 21  1,10 - SNIG
 21  1,10 - SNOG
 21 11,1  - SOG
 21  1,10 - SONG
 20  5,4  - BINO
 20  8,1  | BISONS
 20  8,1  | BISSON
 20  5,4  - BONI
 20  5,2  - GOBI
 20  9,6  | INS
 20  9,6  | ONS
 19  5,14 - BAIS
 19  2,11 - BINGOS
 19  6,6  - BONGS
 19  3,11 - GONGS
 19  2,10 - ING
 19  2,10 - NIG
 19  2,10 - NOG
 19  4,13 - OBJ
 19 13,1  | OBS
 19  9,13 | ONE
 18  5,2  - BINO
 18  8,1  | BISON
 18  5,2  - BONI
 18 13,1  | BOSN
 18  8,12 | GOBI
 18  2,11 - GOINGS
 18  8,12 - ISM
 18  3,11 - NOGGS
 18 11,1  - SIN
 18 11,1  - SON
 17  9,14 | BE
 17  9,14 - BEVER
 17  5,6  | BOZA
 17  4,9  | COIGN
 17 13,1  | GNS
 17  6,13 - JOSIP
 17  3,11 - NOBIS
 17  9,13 | OSE
 17  2,10 - SIG
 17  2,10 - SOG
 16  6,14 - AOB
 16  5,14 - BAI
 16  8,12 | BINI
 16  6,4  - BIO
 16  4,12 | BIOL
 16 13,1  | BIS
 16  7,12 | BO
 16  4,12 | BOIL
 16  8,12 | BONI
 16 13,1  | BOS
 16  8,1  | BOSN
 16  4,9  | CONG
 16  8,12 | GIGI
 16  6,6  - GONGS
 16  6,4  - IBO
 16  9,7  | IS
 16  8,1  | NOSIG
 16  6,4  - OBI
 16  9,3  | ONO
 16  5,6  | ONZA
 16  9,7  | OS
 16  8,3  | SNIB
 16  8,3  | SNOB
 16  8,2  | SSING
 15  4,11 - BOGS
 15  4,9  | COING
 15  4,11 - GIBS
 15 13,1  | GIS
 15  7,12 | GO
 15  4,11 - GOBS
 15 13,1  | INS
 15  6,13 - JIB
 15  6,13 - JOB
 15 13,1  | ONS
 15  5,7  | OZAN
 15  5,8  | ZAGS
 14  7,4  - BI
 14  5,6  - BINGOS
 14  4,11 - BINO
 14  4,11 - BINS
 14  6,2  - BIO
 14  7,4  - BO
 14  8,12 | BOII
 14  4,11 - BONS
 14  8,1  | BOSS
 14  4,11 - GIGS
 14  6,4  - GIO
 14  6,4  - GOI
 14  6,2  - IBO
 14 13,2  | ISN
 14  6,13 - JIG
 14  6,13 - JOG
 14  4,11 - NIBS
 14 13,1  | NIS
 14  7,12 | NO
 14  4,11 - NOBS
 14  6,6  - NOGGS
 14 13,1  | NOS
 14  6,2  - OBI
 14  8,3  | SIBS
 14  8,3  | SIGN
 14  8,3  | SING
 14  8,3  | SNIG
 14  8,3  | SNOG
 14  8,3  | SOBS
 14  8,14 - SOEVER
 14  8,3  | SONG
 14  5,8  | ZAG
 14  5,8  | ZAIN
 14  5,8  - ZETAS
 13  4,11 - BIOS
 13  4,11 - BOIS
 13  4,9  | CIONS
 13  4,9  | COINS
 13  4,11 - GINS
 13  5,6  - GNOSIS
 13  5,6  - GOINGS
 13 10,5  - GORB
 13  7,12 | IO
 13  6,13 - JIN
 13  9,14 | NE
 13  9,14 - NEVER
 13  6,6  - NOBIS
 13  4,11 - NOGS
 13  4,11 - OBIS
 13  7,12 | SO
 12  5,11 - BIO
 12  5,11 - BIS
 12  5,6  - BISONS
 12  5,11 - BOS
 12  4,9  | CION
 12  4,9  | COBS
 12  4,9  | COIN
 12 15,4  | CREDO
 12  7,4  - GI
 12  6,2  - GIO
 12  5,11 - GNS
 12  7,4  - GO
 12  6,2  - GOI
 12  4,11 - GOIS
 12  8,1  | GOSS
 12  5,11 - IBO
 12  6,4  - INO
 12 13,2  | IS
 12 14,8  - NO
 12  5,11 - OBS
 12  9,5  | OI
 12  6,4  - ONI
 12 13,2  | OS
 12 11,1  - SI
 12  8,3  | SIB
 12 11,1  - SO
 12  8,3  | SOB
 11  9,4  - BAT
 11  6,11 - BO
 11  4,9  | COGS
 11  4,9  | COSIN
 11 11,9  - FEODS
 11  5,11 - GIO
 11  5,11 - GIS
 11  4,11 | INSOL
 11  4,11 - IONS
 11  6,13 - JO
 11  9,3  | OB
 11 13,3  | SI
 11  6,6  - SNIBS
 11  6,6  - SNOBS
 11 13,3  | SO
 10  6,14 - AIS
 10  5,14 - BA
 10  5,14 | BI
 10  4,9  | CONI
 10  4,9  | CONS
 10  6,11 - GO
 10  6,2  - INO
 10  5,11 - INO
 10  5,11 - INS
 10  8,12 | IONI
 10  5,11 - NIS
 10  8,12 | NISI
 10  7,4  - NO
 10  4,12 | NOIL
 10  5,11 - NOS
 10  6,2  - ONI
 10  5,11 - ONS
 10  5,14 - SAI
 10  5,14 - SAO
 10  8,3  | SIG
 10  6,6  - SIGNS
 10  8,5  | SINAWA
 10  6,6  - SINGS
 10  8,3  | SINS
 10  8,3  | SION
 10  6,6  - SNIGS
 10  8,3  | SOG
 10  6,6  - SONGS
 10  8,3  | SONS
  9 13,4  - BOC
  9  7,6  - BOGS
  9  8,13 - BOP
  9  4,9  | COB
  9  7,6  - GIBS
  9  7,6  - GOBS
  9  6,11 - NO
  9  8,3  - SOUNDS
  8  7,6  - BINS
  8  7,6  - BONS
  8  4,9  | CGS
  8  4,9  | CIG
  8  4,9  | COG
  8  7,6  - GIGS
  8  8,13 - GIP
  8  7,4  - IO
  8 13,8  | IO
  8  6,11 - IO
  8  6,11 - IS
  8  8,6  | ISAWA
  8  8,2  | ISN
  8  7,6  - NIBS
  8  7,6  - NOBS
  8  7,4  - OI
  8 13,8  | OO
  8  6,11 - OS
  8 13,14 | RG
  8 12,15 - RG
  8  7,4  - SI
  8  8,3  | SIN
  8  7,4  - SO
  8  6,11 - SO
  8  8,3  | SON
  8 12,8  - YIN
  8 12,8  - YON
  7  6,14 - AI
  7  7,2  - BI
  7  7,6  - BIOS
  7  7,2  - BO
  7  7,6  - BOIS
  7  4,13 | BOL
  7  7,6  - BOSS
  7  4,9  | CG
  7  4,9  | CON
  7  7,6  - GINS
  7  9,13 - GP
  7  9,6  | IB
  7  7,14 | IB
  7 13,4  - INC
  7 14,15 - IS
  7  9,4  - NAT
  7  8,13 - NIP
  7  7,6  - NOGS
  7  9,6  | OB
  7  7,6  - OBIS
  7  4,13 | OBL
  7  9,5  - OGOR
  7  9,3  | ON
  7 14,15 - OS
  7  7,6  - SIBS
  7  4,12 | SIOL
  7  7,6  - SOBS
  7  4,12 | SOIL
  6  8,1  | BIS
  6  8,6  - BIS
  6  8,1  | BOS
  6  8,6  - BOS
  6  4,9  | CIS
  6  4,9  | COS
  6  4,9  | CSI
  6  7,2  - GI
  6  4,13 | GIL
  6  8,1  | GNS
  6  8,6  - GNS
  6  7,2  - GO
  6  7,6  - GOIS
  6  4,13 | GOL
  6  7,6  - GOSS
  6  8,13 | OBI
  6  8,1  | OBS
  6  8,6  - OBS
  6 13,4  - OSC
  6 13,4  - SIC
  6 11,1  | SINTO
  6  8,13 - SIP
  6  8,3  | SIS
  6 13,4  - SOC
  6  8,13 - SOP
  6  8,3  | SOS
  6  8,2  | SSI
  6  8,3  | SSI
  5  8,14 | BI
  5  8,1  | GIS
  5  8,6  - GIS
  5  8,13 | GOI
  5  9,6  | IN
  5  7,6  - IONS
  5 15,10 | ISORT
  5  4,13 | NIL
  5  7,2  - NO
  5  4,13 | NOL
  5  9,4  - OAT
  5  9,6  | ON
  5  9,3  | OO
  5  9,13 - OP
  5  9,3  | OS
  5  9,4  - SAT
  5  7,6  - SINS
  5  7,6  - SONS
  5 15,11 | SORTS
  5 15,10 | SSORT
  5  7,8  | TOTO
  5  7,8  | TOTS
  4  8,14 | GI
  4 11,7  - ID
  4  8,1  | INS
  4  8,6  - INS
  4  7,2  - IO
  4  9,6  | IO
  4  9,6  | IS
  4  4,13 | ISL
  4  8,1  | NIS
  4  8,6  - NIS
  4  8,1  | NOS
  4  8,6  - NOS
  4 11,7  - OD
  4  7,2  - OI
  4  9,6  | OI
  4  4,13 | OIL
  4  8,13 | ONI
  4  8,1  | ONS
  4  8,6  - ONS
  4  9,6  | OS
  4  7,2  - SI
  4  8,3  | SI
  4  4,13 | SIL
  4  7,2  - SO
  4  8,3  | SO
  4  4,13 | SOL
  4  7,7  | STOT
  3  8,1  | SIS
  3  8,6  - SIS
  3  8,1  | SOS
  3  8,6  - SOS
  2  8,2  | IS
  2  9,6  - IS
  2  8,14 | OI
  2  8,2  | OS
  2  9,6  - OS
  2  8,14 | SI
