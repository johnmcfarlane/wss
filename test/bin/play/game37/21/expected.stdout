 42  4,4  - BINGO
 26 15,4  | CREDOS
 24  4,2  - BINGO
 21  3,11 - BONGS
 21  4,6  - GIBSONS
 20  3,11 - BIGOS
 20  3,11 - BIOGS
 20  8,1  | BISONS
 20  9,6  | INS
 20  9,6  | ONS
 19  2,11 - BINGOS
 19  2,11 - BOINGS
 19  6,6  - BONGS
 19  3,11 - GONGS
 19  9,13 | ONE
 18  6,6  - BIGOS
 18  6,6  - BIOGS
 18  8,1  | BISON
 18  2,11 - GOINGS
 18  3,11 - NOGGS
 17  4,9  | COIGN
 17  9,13 | OSE
 16  6,4  - BIO
 16 13,1  | BIS
 16  7,12 | BO
 16  4,12 | BOIL
 16 13,1  | BOS
 16  6,6  - GONGS
 16  9,7  | IS
 16  6,4  - OBI
 16  9,7  | OS
 16  8,3  | SNIB
 16  8,3  | SNOB
 15  4,11 - BIGS
 15  4,11 - BOGS
 15  4,11 - GIBS
 15  7,12 | GO
 15  4,11 - GOBS
 15 13,1  | INS
 15  6,13 - JIB
 15  6,13 - JOB
 15 13,1  | ONS
 15  5,8  | ZAGS
 14  7,4  - BI
 14  5,6  - BINGOS
 14  4,11 - BINS
 14  6,2  - BIO
 14  7,4  - BO
 14  5,6  - BOINGS
 14  8,1  | BOSS
 14  4,11 - GIGS
 14  4,11 - IGGS
 14  6,13 - JIG
 14  6,13 - JOG
 14  4,11 - NIBS
 14  7,12 | NO
 14  4,11 - NOBS
 14  6,6  - NOGGS
 14 13,1  | NOS
 14  6,2  - OBI
 14  8,3  | SIBS
 14  8,3  | SIGN
 14  8,3  | SING
 14  8,3  | SNOG
 14  8,3  | SOBS
 14  8,14 - SOEVER
 14  8,3  | SONG
 14  5,8  | ZAG
 14  5,8  - ZETAS
 13  4,11 - BIOS
 13  4,9  | CIONS
 13  4,9  | COINS
 13  4,11 - GINS
 13  5,6  - GNOSIS
 13  5,6  - GOINGS
 13  6,13 - JIN
 13  9,14 | NE
 13  9,14 - NEVER
 13  4,11 - NOGS
 13  4,11 - OBIS
 13  7,12 | SO
 12  5,11 - BIS
 12  5,6  - BISONS
 12  5,11 - BOS
 12  4,9  | CION
 12  4,9  | COBS
 12  4,9  | COIN
 12 15,4  | CREDO
 12  7,4  - GO
 12 13,2  | IS
 12 14,8  - NO
 12 13,2  | OS
 12  8,3  | SIB
 12  8,3  | SOB
 12  5,8  | ZAS
 11  4,9  | CIGS
 11  4,9  | COGS
 11 11,9  - FEODS
 11  4,11 - IONS
 11  6,13 - JO
 11 13,3  | SI
 11  6,6  - SNIBS
 11  6,6  - SNOBS
 11 13,3  | SO
 10  6,14 - AIS
 10  5,14 - BA
 10  5,14 | BI
 10  4,9  | CONI
 10  4,9  | CONS
 10  5,11 - INS
 10  8,12 | NISI
 10  7,4  - NO
 10  4,12 | NOIL
 10  5,11 - NOS
 10  5,11 - ONS
 10  6,6  - SIGNS
 10  6,6  - SINGS
 10  8,3  | SINS
 10  6,6  - SNOGS
 10  6,6  - SONGS
 10  8,3  | SONS
  9  7,6  - BIGS
  9  7,6  - BOGS
  9  8,13 - BOP
  9  4,9  | COB
  9  7,6  - GIBS
  9  7,6  - GOBS
  9  8,3  - SOUNDS
  8  7,6  - BINS
  8  4,9  | CIG
  8  4,9  | COG
  8  7,6  - GIGS
  8  8,13 - GIP
  8  7,6  - IGGS
  8  6,11 - IS
  8  7,6  - NIBS
  8  7,6  - NOBS
  8  7,4  - OI
  8  6,11 - OS
  8  7,4  - SI
  8  8,3  | SIN
  8  7,4  - SO
  8  8,3  | SON
  8  7,7  | STOTIN
  7  6,14 - AI
  7  7,2  - BI
  7  7,6  - BIOS
  7  7,2  - BO
  7  7,6  - BOSS
  7  4,9  | CON
  7  7,6  - GINS
  7  9,6  | IB
  7  7,14 | IB
  7 14,15 - IS
  7  8,13 - NIP
  7  7,6  - NOGS
  7  7,6  - OBIS
  7 14,15 - OS
  7  7,6  - SIBS
  7  7,6  - SOBS
  7  4,12 | SOIL
  6  8,1  | BIS
  6  8,6  - BIS
  6  8,1  | BOS
  6  8,6  - BOS
  6  4,9  | CIS
  6  4,9  | COS
  6  7,2  - GO
  6  8,13 | OBI
  6 13,4  - SIC
  6  8,13 - SIP
  6  8,3  | SIS
  6  8,13 - SOP
  6  8,3  | SOS
  5  8,14 | BI
  5  9,6  | IN
  5  7,6  - IONS
  5  4,13 | NIL
  5  7,2  - NO
  5  9,6  | ON
  5  9,13 - OP
  5  9,3  | OS
  5  9,4  - SAT
  5  7,6  - SINS
  5  7,6  - SONS
  5 15,11 | SORTS
  5  7,8  | TOTS
  4 11,7  - ID
  4  8,1  | INS
  4  8,6  - INS
  4  9,6  | IS
  4  8,1  | NOS
  4  8,6  - NOS
  4 11,7  - OD
  4  7,2  - OI
  4  9,6  | OI
  4  4,13 | OIL
  4  8,1  | ONS
  4  8,6  - ONS
  4  9,6  | OS
  4  7,2  - SI
  4  8,3  | SI
  4  7,2  - SO
  4  8,3  | SO
  4  4,13 | SOL
  4  7,7  | STOT
  3  8,1  | SIS
  3  8,6  - SIS
  3  8,1  | SOS
  3  8,6  - SOS
  2  8,2  | IS
  2  9,6  - IS
  2  8,14 | OI
  2  8,2  | OS
  2  9,6  - OS
  2  8,14 | SI
